File: Objetos/objeto_jogada.js
//Objeto Jogada com os atributos peça, casa, peça eliminada
class Jogada{
    constructor(id_jogada, peca, casa, pecaEliminada){
        this.id_jogada = id_jogada;
        this.peca = peca;
        this.casa = casa;
        this.pecaEliminada = pecaEliminada;
    }

    returnID_jogada(){
        return this.id_jogada;
    }

    returnPeca(){
        return this.peca;
    }

    returncasa(){
        return this.casa;
    }

    returnPecaeliminada(){
        return this.pecaEliminada;
    }
}

export default {Jogada};

File: Objetos/objeto_jogada.py
#Objeto Jogada com os atributos peça, casa, peça eliminada
class Jogada:
    def __init__(self, id_jogada, time, peca, casa, pecaEliminada):
        self.id_jogada = id_jogada
        self.time = time
        self.peca = peca
        self.casa = casa
        self.pecaEliminada = pecaEliminada


    def returnID_jogada(self):
        return self.id_jogada
    
    def returnTime(self):
        return self.time

    def returnPeca(self):
        return self.peca
    

    def returncasa(self):
        return self.casa
    

    def returnPecaeliminada(self):
        return self.pecaEliminada
    